fs/mmap/fs_anonmap.c

```diff
@@ -39,32 +39,69 @@ static int unmap_anonymous(FAR struct task_group_s *group,
                            FAR void *start,
                            size_t length)
 {
-  int ret;
+  FAR void *newaddr;
```

Comment:
```suggestion
  FAR void *newaddr unused_data;
```
nit: An alternative to the `UNUSED(newaddr)` in line 103 is the `unused` attribute, which indicates that the variable might not be used, preventing a compiler warning.